docs/developer/extending/apps/synchronous-webhooks/key-concepts.mdx

```diff
@@ -26,6 +26,8 @@ During HTTPS requests there are several headers included:
 - `Saleor-Domain` - defines a Saleor domain
 - `Saleor-Signature` - defines the signature to indicate that the request is verifiable
 
+Saleor gives you an option to add custom headers to the request (available since version 3.12). Currently supports `Authorization*` and `X-*` keys with limitation of 5 headers per webhook and 998 characters per header. To check how to add `customHeaders` go to [Subscribing to webhook](/docs/developer/extending/apps/asynchronous-webhooks.mdx#subscribing-to-a-webhook).
+
```

Comment:
```suggestion
Saleor gives you an option to add custom headers to the request (available since version 3.12). Currently supports `Authorization*` and `X-*` keys with limitations of 5 headers per webhook and 998 characters per header. To check how to add `customHeaders` go to [Subscribing to webhook](/docs/developer/extending/apps/asynchronous-webhooks.mdx#subscribing-to-a-webhook).

```